tools/run_tests/run_tests.py

```diff
@@ -1591,7 +1591,8 @@ def make_jobspec(cfg, targets, makefile='Makefile'):
             jobset.JobSpec(
                 [
                     'cmake', '--build', '.', '--target',
-                    '%s' % target, '--config', _MSBUILD_CONFIG[cfg]
+                    '%s' % target, '--config', _MSBUILD_CONFIG[cfg],
+                    '--', '/maxcpucount:%d' % args.jobs,
```

Comment:
does `-- /maxcpucount:%d' imply that the argument is passed directly to msbuild?

my concern is that might not work if other generators are used (e.g. Ninja).  Isn't there a standard way to set parallelism in cmake?